File: js/1-TwoSum.js
// 2022-3-29 20:46
// Runtime 116ms(27.7%)
// Memory 41.7mb(43.8%)

/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var find = false;
    for(var i = 0; i < nums.length; i ++) {
        if(find) break;
        for(var j = i + 1; j < nums.length; j++) {
            if(nums[i] + nums[j] == target) {
                var ret = [i, j];
                find = true;
                break;
            }
        }
    }
    return ret;
};

File: csharp/1-TwoSum.cs
// 2022-3-29 20:38
// Runtime 160ms(53.5%)
// Memory 43mb(63.3%)

public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int[] ret = new int[2];
        bool find = false;
        for(int i = 0; i < nums.Length; i = i + 1) {
            if(find) break;
            for(int j = i + 1; j < nums.Length; j = j + 1) {
                if(nums[i] + nums[j] == target) {
                    ret[0] = i;
                    ret[1] = j;
                    find = true;
                    break;
                }
            }
        }
        return ret;
    }
}
